static-build/nemu/Dockerfile_aarch64

```diff
@@ -0,0 +1,64 @@
+FROM debian
+
+ARG NEMU_REPO
+ARG NEMU_VERSION
+ARG NEMU_OVMF
+ARG VIRTIOFSD_RELEASE
+ARG VIRTIOFSD
+ARG PREFIX
+
+WORKDIR /root/nemu
+
+RUN echo "deb http://deb.debian.org/debian stable main" > /etc/apt/sources.list
+RUN echo "deb http://deb.debian.org/debian stable-updates main" >> /etc/apt/sources.list
+RUN echo "deb http://security.debian.org stable/updates main" >> /etc/apt/sources.list
+RUN dpkg --add-architecture arm64
+
+RUN apt update && apt install -y \
+	gcc-aarch64-linux-gnu \
+	autoconf \
+	automake \
+	bc \
+	bison \
+	cpio \
+	flex \
+	gawk \
+	pkg-config \
+	python \
+	python-dev \
+	rsync \
+	wget \
+	git \
+	make \
+	libaudit-dev:arm64 \
+	libcap-dev:arm64 \
+	libcap-ng-dev:arm64 \
+	libdw-dev:arm64 \
+	libelf-dev:arm64 \
+	libglib2.0-0:arm64 \
+	libglib2.0-dev:arm64 \
+	zlib1g-dev:arm64 \
+	librbd-dev:arm64 \
+	libltdl-dev:arm64 \
+	libpixman-1-dev:arm64
+
+RUN cd  .. && git clone --depth=1 "${NEMU_REPO}" nemu
+RUN git fetch origin --tags && git checkout "${NEMU_VERSION}"
+RUN git clone https://github.com/qemu/capstone.git capstone
+RUN git clone https://github.com/qemu/keycodemapdb.git ui/keycodemapdb
+ADD configure-hypervisor.sh /root/configure-hypervisor.sh
+RUN wget http://archive.ubuntu.com/ubuntu/pool/universe/g/gcc-defaults/pkg-config-aarch64-linux-gnu_7.4.0-1ubuntu2.3_amd64.deb && dpkg -i pkg*
+
+RUN cp -r /usr/lib/aarch64-linux-gnu/pkgconfig/ /usr/aarch64-linux-gnu/lib/
+
+RUN PREFIX="${PREFIX}" /root/configure-hypervisor.sh -s kata-nemu aarch64 | xargs ./configure \
+       --with-pkgversion=kata-static
+RUN make -j$(nproc) CONFIG_HID_DEV=y
```

Comment:
If this is only going to run under the CI, this is fine.

However, if this is going to run on a developers system, this is going to DoS it for the duration of the `make` run.

In the latter case, we tend to do:

```sh
make -j$(($(nproc)-1)) ...
```

Reply:
very appreciate @jodh-intel . I have refined this patch base on most of your precious feedback.  Further more I try to make this cross-compile can work on other arch as your opinion says, but only test on arm64.